sap/5.2/modules/ROOT/pages/sap-connector-reference.adoc

```diff
@@ -0,0 +1,638 @@
+| Reconnection a| <<Reconnection>> |  When an application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
+|===
+[[sap_simple-connection-provider]]
+===== Simple Connection Provider
+
+
+Connection provider that connects using a username and password.
+
+====== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| SAP system number a| String |  The two-digit system number (sysnr) of the SAP system. |  | x
+| SAP client ID a| String |  Three-digit client number. Leading zeros are preserved if they appear in the number. |  | x
+| Destination Name a| String |  Identifier of the destination generated by this connector. If none is set, then a random value is created. |  |
+| Login language a| String |  The language code used by the connector. The language determines the code page used for communicating between the SAP Adapter and the application server. |  EN |
+| Extended Properties a| Object |  Use to pass in all the database server-specific connection parameters. |  |
+| Username a| String |  The username the connector uses to log in to SAP. |  | x
+| Password a| String |  The password associated with the login username. |  | x
+| Application server host a| String |  The host of the SAP application server. |  |
+| Message server configuration a| <<MessageServer>> |  The configuration for message server usage. |  |
+| Enable RFC tracing on server a| Boolean |  Enable or disable RFC trace on the server. |  `false` |
+| Enable CPIC tracing on server a| Enumeration, one of:
+
+** INHERIT
+** NO_TRACING
+** LEVEL_1
+** LEVEL_2
+** LEVEL_3 |  Enable or disable RFC trace on the server. |  |
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
+|===
+
+== Sources
+
+* <<document-listener>>
+* <<function-listener>>
+
+[[document-listener]]
+=== Document Listener
+`<sap:document-listener>`
+
+Source that listens for incoming IDocs.
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Gateway Host a| String |  Name of the host running the gateway server. |  | x
+| Gateway Service a| String |  Either the name or service port for the gateway service of the SAP system. |  | x
+| Program ID a| String |  SAP system program ID that is registered on the gateway. Must be unique for the given gateway. |  | x
+| Connection Count a| Number |  The number of connections to register on the gateway. |  1 |
+| Idoc Type Filter Regex a| String |  Expression to filter by type, all incoming IDocs. |  |
```

Comment:
please add this below :

| IDoc Render Options a| <<`IDocRenderParameterGroup`>> |  +++IDoc Rendering options+++ |  |